assets/js/script.js

```diff
@@ -0,0 +1,127 @@
+const closeButton = document.querySelector('.close');
+closeButton.addEventListener('click', () => {
+  document.querySelector('.mobile').classList.toggle('show');
+});
+
+const navButton = document.querySelector('#nav-button');
+navButton.addEventListener('click', () => {
+  document.querySelector('.mobile').classList.toggle('show');
+});
+
+let showAll = false;
+const seeMoreBtn = document.getElementById('see-more-btn');
+// const additionalContent = document.querySelectorAll('.show-mobile');
```

Comment:
- [ ] Please ensure that you remove the commented unused code in your code base, Commented code can clutter your repository and make it harder for others (and even yourself) to read and understand the active codebase. It can create confusion and increase the cognitive load when navigating the project.